src/orion/algo/hyperband.py

```diff
@@ -78,14 +108,19 @@ class Hyperband(BaseAlgorithm):
     seed: None, int or sequence of int
         Seed for the random number generator used to sample new trials.
         Default: ``None``
+    frequency: int
```

Comment:
I believe `repetitions` would be more accurate.

Reply:
You always have the good name :)